src/components/Speciality/HowWeWorkWith.js

```diff
@@ -0,0 +1,54 @@
+import React, { Fragment } from 'react'
+import styled from 'styled-components'
+import { Row, Col, Grid } from '../grid'
+import breakpoint from 'styled-components-breakpoint'
+
+import GreyBackground from '../Common/GreyBackground'
+import { SectionTitle, BodyPrimary } from '../Typography'
+import Hr from '../Common/Hr'
+
+const StyledRow = styled(Row)`
+  padding-top: ${({ theme }) => theme.space[5]};
+  padding-bottom: ${({ theme }) => theme.space[5]};
+
+  ${breakpoint('smallTablet')`
+    padding-top: ${({ theme }) => theme.space[6]};
+    padding-bottom: ${({ theme }) => theme.space[6]};
+    `}
+
+  ${breakpoint('tablet')`
+    padding-top: ${({ theme }) => theme.space[7]};
+    padding-bottom: ${({ theme }) => theme.space[7]};
+  `}
+`
+
+const HowWeWorkWithSection = ({
+  howWeWorkWithTitle,
+  howWeWorkWithCopy,
+  howWeWorkWithPractises
+}) => {
+  return (
```

Comment:
Just a small detail but the `return` does not need to be there. `=> (` automatically returns